l10n_it_account_balance_report/report/account_balance_report.py

```diff
@@ -0,0 +1,292 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools.float_utils import float_compare, float_is_zero
+from odoo.tools.pycompat import string_types
+from odoo.tools.safe_eval import safe_eval
+
+
+def get_xmlid(id_str):
+    id_str = id_str.replace('.', '')
+    return 'l10n_it_account_balance_report.{}'.format(id_str)
+
+
+class ReportAccountBalanceReport(models.TransientModel):
+    _name = 'account_balance_report'
```

Comment:
Non è usato da nessun'altra parte un model con questo nome?

Reply:
@eLBati non nei moduli standard di Odoo e nel repo OCA/account-financial-reporting (che ho usato come benchmark). Ad ogni modo, dovessero sorgere poi delle incompatibilità, potremmo cambiare anche `_name` in corsa senza rischio di perdere dati visto che sono `TransientModel`, no?

Reply:
Direi di sì